web/pandas_web.py

```diff
@@ -65,7 +67,30 @@ class Preprocessors:
     """
 
     @staticmethod
-    def current_year(context):
+    def process_translations(context: dict) -> dict:
```

Comment:
I think `download_translated_docs_and_add_translated_navbar` would be a bit more descriptive. Or maybe we can have two separate preprocessors. It'd also be helpful to add more context to the docstring. Like, explaining that the translations are done in a separate repo, that we download the whole repo tar even if we only need the language directories, that the site generator will render the translated pages as the rest as we copy them in the website directory... Anyone having to deal with this code will appreciate all this information in the future.